automatic/wps-office-free/wps-office-free.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>wps-office-free</id>
+    <version>10.1.0.5507</version>
+    <title>WPS Office 2016 Free</title>
+    <owners>chocolatey</owners>
+    <authors>Kingsoft Software</authors>
+    <projectUrl>http://www.ksosoft.com/product/office-free</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/e4a49519947c3cff55c17a0b08266c56b0613e64/icons/kingsoft-office-free.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description><![CDATA[
+<p style="text-align: center;">WPS Office 2016 Free is a free office productivity suite which is smaller and faster <span>than most other office suites. <br>The latest WPS Office 2016 Free allows you to<a href="http://www.kingsoftstore.com/news/3324-mutli-language-office.html" target="_blank">switch language UI</a> and access files on different devices by using <br></span><a href="http://help.wps.com/customer/portal/articles/1641528-how-to-use-file-roaming-in-wps-office" target="_blank">File Roaming </a> A brand new Docer Online Templates page is also added. See<a style="line-height: 1.3em;" title="what is new in WPS Office Free" href="http://www.ksosoft.com/what-is-new-in-kingsoft-office-free-2013.html" target="_blank"> what's new</a>.</p>
+    ]]></description>
+    <summary>WPS Office Free is a free office productivity suite which is smaller and faster than most other office suites.</summary>
+    <language>en-US</language>
+    <tags>kingsoft WPS Office admin</tags>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
```

Comment:
Package source must point to the package root directory, not the repo root directory as per [§1.2.1](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#121-obligatory-metadata)